package/firmware/ipq-wifi/Makefile

```diff
@@ -27,7 +27,8 @@ ALLWIFIBOARDS:= \
 	aruba_ap-303 \
 	avm_fritzrepeater-1200 \
 	linksys_ea8300 \
-	qxwlan_e2600ac
+	qxwlan_e2600ac \
+	ezviz_cs-w3-wd1200g-eup
```

Comment:
Please use correct alphabetic sorting for positioning, same goes for several cases below.